ui/shared/components/panel/family/Family.jsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react'
+import PropTypes from 'prop-types'
```

Comment:
identical to `family.jsx` but with a bunch of helper components moved to other files